plugins/flatpak/gs-flatpak.c

```diff
@@ -1760,9 +1737,10 @@ gs_plugin_refine_item_state (GsFlatpak *self,
 			     GCancellable *cancellable,
 			     GError **error)
 {
-	guint i;
 	g_autoptr(GPtrArray) xrefs = NULL;
 	g_autoptr(AsProfileTask) ptask = NULL;
+	g_autoptr(FlatpakInstalledRef) ref = NULL;
```

Comment:
Perhaps you could name this one `xref`, for consistency with the rest of the code (and the code being removed)

Reply:
I think the _xref_ is legacy from when Flatpak was called xdg-app... But I can rename it...